django-server/server/server/.env.sample

```diff
@@ -1,5 +0,0 @@
-USE_POSTGRES=False
```

Comment:
Why is this file deleted?

Reply:
as for me it was a sample how to configure  the env on server and is no longer needed

Reply:
.env is no longer needed? If it is needed, then why delete .sample?

Reply:
env is defined by the environmnent variables on heroku in system. sample back again to repo